src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -123,10 +127,11 @@ class AirspeedModule : public ModuleBase<AirspeedModule>, public ModuleParams,
 	uORB::Subscription _vehicle_land_detected_sub{ORB_ID(vehicle_land_detected)};
 	uORB::Subscription _vehicle_local_position_sub{ORB_ID(vehicle_local_position)};
 	uORB::Subscription _vehicle_status_sub{ORB_ID(vehicle_status)};
-	uORB::Subscription _vtol_vehicle_status_sub{ORB_ID(vtol_vehicle_status)};
+	uORB::Subscription _vehicle_thrust_setpoint_0_sub{ORB_ID(vehicle_thrust_setpoint), 0};
```

Comment:
It's also instance 0 in the VTOL plane case, right?

Reply:
Yes.